packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss

```diff
@@ -1,80 +1,78 @@
-  }
+.root {
+  -webkit-font-smoothing: antialiased;
+  background-color: $ms-color-white;
+  border: 1px solid $ms-color-neutralLight;
+  box-sizing: border-box;
+  max-width: 320px;
+  min-width: 206px;
+  user-select: none;
+  position: relative;
+}
 
-  .ms-DocumentCard--actionable:hover {
-    cursor: pointer;
-    border-color: $ms-color-neutralTertiaryAlt;
+.isRootActionable:hover {
+  cursor: pointer;
+  border-color: $ms-color-neutralTertiaryAlt;
 
-    // Place a heavier border within the document card
-    &:after {
-      content: '';
-      position: absolute;
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
-      border: 1px solid $ms-color-neutralTertiaryAlt;
-      pointer-events: none;
-    }
+  // Place a heavier border within the document card
+  &:after {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    border: 1px solid $ms-color-neutralTertiaryAlt;
+    pointer-events: none;
   }
+}
 
-  // When a DocumentCardTitle follows a DocumentCardLocation, we want to reduce the padding
-  .ms-DocumentCardLocation + .ms-DocumentCardTitle {
-    padding-top: 4px;
-  }
+// When a DocumentCardTitle follows a DocumentCardLocation, we want to reduce the padding
+.location + .title {
+  padding-top: 4px;
+}
 
-  // Modifier: Compact layout
-  .ms-DocumentCard--compact {
-    border-bottom: 2px solid $ms-color-neutralTertiary; // Default border color
-    display: flex;
-    max-width: 480px;
-    height: 109px;
+// Modifier: Compact layout
+.isRootCompact {
+  border-bottom: 2px solid $ms-color-neutralTertiary; // Default border color
+  display: flex;
+  max-width: 480px;
+  height: 109px;
 
-    // Remove the usual accent color from the preview
-    .ms-DocumentCardPreview {
-      border-bottom: none;
-      max-height: 106px;
-      max-width: 144px;
+  // Remove the usual accent color from the preview
+  .preview {
+    border-bottom: none;
+    max-height: 106px;
+    max-width: 144px;
 
-      .ms-DocumentCardPreview-icon {
-        .ms-Image-image {
-          max-height: 32px;
-          max-width: 32px;
-        }
+    .icon {
+      :global(.ms-Image-image) {
+        max-height: 32px;
+        max-width: 32px;
       }
     }
+  }
 
-    .ms-DocumentCardPreview {
-      @include border-right(1px, solid, $ms-color-neutralLight);
-    }
+  .preview {
+    @include border-right(1px, solid, $ms-color-neutralLight);
+  }
 
-    // Expand the details area to fill the remaining width
-    .ms-DocumentCard-details {
-      display: flex;
-      flex-direction: column;
-      flex: 1;
-      justify-content: space-between;
-    }
+  // Expand the details area to fill the remaining width
+  .details {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    justify-content: space-between;
+  }
 
-    .ms-DocumentCardTitle {
-        padding: 12px 16px 8px 16px;
-        @include ms-font-m;
-        line-height: 16px;
-    }
+  .title {
```

Comment:
Hmm, `DocumentCardTitle` is a separate component from `DocumentCard`. This is going to require a different approach because we have parent components overriding the styles of children, which we can no longer do with CSS modules.